test/functorch/test_control_flow.py

```diff
@@ -1238,11 +1258,57 @@ def mul(x: torch.Tensor, y: torch.Tensor):
             )
         self.assertEqual(cumsum1, cumsum_exp)
 
+    def test_pointwise_scan_simple_graph(self):
+        def add(x: torch.Tensor, y: torch.Tensor):
```

Comment:
Do we support combine functions that takes pytree inputs? e.g.
```
def combine(a_dict_list, b_dict_list):
  ...
```
If not we should state this clearly in the linline doc of scan.

Reply:
No, we support pytree inputs as well. Let me know if this is not well enough stated in the docstring of scan.